PowerShell Scanners/Get-ChocolateyPackages/README.md

```diff
@@ -0,0 +1,14 @@
+# Description
+
+Queries a machine for Chocolatey Package information. If using FOSS choco, you get Name and Version, if licensed you get a full audit.
+
+# Requirements
+
+* A version of choco to be installed on the workstation/server
```

Comment:
This probably doesn't need to be a list since it's just 1 item 😃 .

Reply:
You're probably right 😂